action.yml

```
@@ -77,6 +77,18 @@ inputs:
       Skip tests and code coverage data processing.
       Set to 'true'.
     required: false
+  skip-rosdep-install:
+    default: ""
+    description: |
+      Skip rosdep install.
+      Set to 'true'.
+    required: false
+  rosdep-check:
+    default: ""
+    description: |
+      Check ros dependency when use with skip-rosdep-install.
```

To make this a bit clearer:

```suggestion
      Check dependencies of packages using 'rosdep check'. Must be used with skip-rosdep-install.
```